### DISCUSSION TITLE
How to make a RightDrawerContent?

### DISCUSSION THREAD

**Discussion by hxhgxy** [MaterialDesignInXAML/MaterialDesignInXamlToolkit#3255] [Q&A] [answered]
In MaterialDesign demo source, I try to change the LeftDrawerContent to RightDrawerContent, the drawer doesn't show. What else i should to do?

thanks,
Tiger

**Comment by nicolaihenriksen** [comment, marked as answer]
@hxhgxy If I understand your question correctly, you simply want the `DrawerHost` to open up the right drawer instead of the left drawer?

Setting the `RightDrawerContent` only tells the drawer host what should be displayed if/when the right drawer is opened. You also need something to actually trigger the opening. In the demo app (MainWindow) this is bound to the `MenuToggleButton` and the state of that is in turn bound to the `DrawerHost.IsLeftDrawerOpen` which effectively means when you toggle the button, it will show the left drawer. So you need to change BOTH the content and the "trigger", to get what I think you want. These changes to MainWindow.xaml should do the trick:

![image](https://github.com/MaterialDesignInXAML/MaterialDesignInXamlToolkit/assets/19572699/f544c186-b52b-4227-a237-13183d73a83d)

**Comment by hxhgxy** [comment]
oh, yes, i fotgot to change the IsLeftDrawerOpen to IsRightDrawerOpen...
thank you!
